tmail-backend/apps/distributed-es6-backport/src/test/java/com/linagora/tmail/james/app/DistributedServerTest.java

```diff
@@ -60,4 +63,10 @@ public void shouldUseCassandraMailboxManager(GuiceJamesServer jamesServer) {
         assertThat(jamesServer.getProbe(MailboxManagerClassProbe.class).getMailboxManagerClass())
             .isEqualTo(CassandraMailboxManager.class);
     }
+
+    @Test
+    public void shouldUseCassandraUsersDAOAsDefault(GuiceJamesServer jamesServer) {
```

Comment:
Maybe should use UserRepository is better than UserDAO to check. 
But we haven't  `CassandraUsersRepository`  dedicated class. 
Right now, we use anonymous 
`bind(UsersRepository.class).to(new TypeLiteral<UsersRepositoryImpl<CassandraUsersDAO>>() {});`

Reply:
it's alright for me the way it is actually :)